app/models/lettings_log.rb

```diff
@@ -533,7 +533,9 @@ def managing_organisation_name
   def beds_for_la_rent_range
     return 0 if is_supported_housing?
 
-    beds.nil? ? nil : [beds, LaRentRange::MAX_BEDS].min
+    real_beds = beds || (1 if is_bedsit?)
```

Comment:
This has both and if and an or, I think it's a bit confusing to read. Would it make sense to change it to `real_beds = is_bedsit? ? 1 : beds` ?
Also, real_beds kinda implies there's a fake_beds, is there a better name for it? The only thing that came to mind for me was current_beds, but it's up to you

Reply:
I'm now realising it would be a different statement, but I assume there's no reason to prioritise beds before 1 for bedsit (?)

Reply:
yeah I can't think of a particularly spectacular name since the reason for this code is just general convoluted grossness.
beds_accounting_for_it_being_cleared_and_then_derived but I think that might be overkill

I think reordering it into a ternary is fine, but it makes no difference and is a matter of taste. I gently prefer what I've done since I feel like it reads that by default it should be beds, but then if that's nil we'll check if it should be something.

Reply:
In that case could we not contribute to the general convoluted grossness and get the value to come in as the correct value to begin with?

I think it might not get cleared if we add this to the beds question:
 ```
 def derived?(log)
    log.is_bedsit?
  end
 ```

Let me know if that works tho

Reply:
this is much cleaner, thanks for pointing me towards this.